includes/Post/DeletePost.php

```diff
@@ -91,7 +91,7 @@ function DeletePost ( $DeleteThisPost)
 
 		$stmt->execute();
 		
-		$stmt = $sqlConn->prepare ("DELETE FROM $assignPostTable WHERE $postTitle REGEXP \"^$userN\"");
+		$stmt = $sqlConn->prepare ("DELETE FROM $assignPostTable WHERE $postTitle REGEXP \"^$userN$\"");
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space before opening parenthesis of function call prohibited

<!-- Comment by [SideCI](https://sideci.com) -->